### PR TITLE
Update E2E tests for Twenty Twenty theme

### PR DESCRIPTION
**Pull request by noisysocks**
Update the E2E tests to work with the new Twenty Twenty theme in trunk.

- [x]  ● Heading › it should correctly apply named colors
- [x]  ● TypeWriter › should maintain caret position after scroll
- [x]  ● TypeWriter › should scroll caret into view from the top
- [x]  ● Font Size Picker › should apply a named font size using the font size input
- [x]  ● Preview with Custom Fields enabled › displays edits to the post title and content in the preview
- [x]  ● Managing reusable blocks › Should import reusable blocks

### CODE REVIEW

**Review by talldan — APPROVED** [review]
Great work on this!

File: packages/e2e-tests/specs/manage-reusable-blocks.test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe( 'Managing reusable blocks', () => {
 	 * @return {Promise} Promise resolving to number of post list entries.
 	 */
 	async function getNumberOfEntries() {
-		return page.evaluate( () => document.querySelectorAll( '.entry' ).length );
+		return page.evaluate( () => document.querySelectorAll( '.hentry' ).length );
```
Comment (gziolo) [inline packages/e2e-tests/specs/manage-reusable-blocks.test.js]: `.hentry`? 🙃

**Comment by gziolo** [comment]
Congrats for making Travis happy again 💯

File: packages/e2e-tests/specs/manage-reusable-blocks.test.js (hunk 1 above)
Comment (noisysocks) [reply packages/e2e-tests/specs/manage-reusable-blocks.test.js]: Yeah I'm not really sure how this ever worked. The rows in the Manage All Blocks screen have class `hentry`. We could maybe consider selecting `.type-wp_block` instead.

Screen Shot 2019-09-25 at 16 17 24

**Comment by ellatrix** [comment]
What typewriter tests were failing?

**Comment by noisysocks** [comment]
> What typewriter tests were failing?

These two:

- TypeWriter › should maintain caret position after scroll
- TypeWriter › should scroll caret into view from the top

Both failures were to do with the default font size changing. This meant that there wasn't a scrollable container on the page. Would appreciate you double checking the changes in this PR to make sure I didn't bugger it up 🙂

**Comment by ellatrix** [comment]
@noisysocks I'm not sure if they have to do with the font size change. Some of the tests are now changed and don't test the correct thing anymore. E.g. the scroll test should happen at the very start, not after the page has already been scrolled. Something is going wrong because in the e2e test there is no scroll container.

**Comment by ellatrix** [comment]
Screenshot 2019-09-26 at 19 09 56

The container has now a much smaller width, so because of the styles it's not scrollable.

**Comment by ellatrix** [comment]
I'm not sure why that whitespace is there...

**Comment by noisysocks** [comment]
Thanks for checking! Looks like I did indeed bugger it up 🙂

That margin is very strange. Does it appear when you run Gutenberg normally? Is it something to do with the WP Admin menu being collapsed?

**Pull request merged**